### DISCUSSION TITLE
Buttons go off the div in mobile (for you)

### DISCUSSION THREAD

**Discussion by hslee2008** [community/community#16242] [Other Feature Feedback, Questions, & Ideas] [answered]
The emojis go off the div.
It is not a huge problem but it annoying.
Please fix this as soon as possible.

![Screenshot_20220509-185512_Brave - Nightly](https://user-images.githubusercontent.com/78584173/167386973-684a14cf-3665-4044-b5fb-882db065bb11.jpg)

**Comment by mxie** [comment]
Thanks for your feedback @HyunseungLee-Travis!

Are you experiencing this in a mobile browser or in one of our mobile apps?

**Reply by hslee2008** [reply]
Only in github.com website (mobile)

**Reply by mxie** [reply]
Got it - thanks! I'm going to transfer this over to the Feeds category since it's about the mobile apps.

**Comment by ghost** [comment]
Rack you

**Comment by hslee2008** [comment, marked as answer]
![Screenshot_20220715-223154_Brave - Nightly.png](https://user-images.githubusercontent.com/78584173/179233367-9e29ef96-f243-42e7-9e95-dd33af494031.png)

I guess it is fixed now.

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
